hr_expense_petty_cash/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_hr_expense_petty_cash,access_hr_expense_petty_cash,model_petty_cash,account.group_account_invoice,1,1,1,1
```

Comment:
Perhaps only the manager should be able to change these, in accordance with the rest of the settings.